doc/manual.asciidoc

```diff
@@ -861,7 +861,13 @@ one_word_with_no_space = foo$
 Other whitespace is only significant if it's at the beginning of a
 line.  If a line is indented more than the previous one, it's
 considered part of its parent's scope; if it is indented less than the
-previous one, it closes the previous scope.
+previous one, it closes the previous scope. Note that the physical line that is
+a line continuation is not considered as a separate line for the scope
+determination purposes.
+
+_Available since Ninja 1.12._
+Horizontal Tab character (0x09) can be used for indentation. Earlier Ninja
+versions allow only space (0x20) character to be used for such purpose.
```

Comment:
Hm ... while technically correct, I find these unnecessary. Every programmer (that is writing Ninja files by hand) should know what "tabs" and "spaces" are. I would simplify it:

"Horizontal Tab character (0x09)" -> Tabs
"space (0x20) character" -> spaces

Opinions?

Reply:
sure